.github/ISSUE_TEMPLATE/--design-related.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Hmm that's weird, I think it is there because I added an emoji to the title, to matke it match the other templates that had emojis at the front, and it must have converted the emoji + space to dashes.

Reply:
If you look at the other files, they have the emoji in the name inside the table but not in the filename, so I think the filename essentially doesn't matter.

The other files have the emoji in the name entry and the title is blank — I don't know how that difference works but I thought it might be worth flagging.

Reply:
huh, weird. I'm also not so bothered by this, but it is funny. i might try deleting `--` and seeing what happens, if it all works fine without then i'd prefer not to have for consistency reasons